lib/facebook.ex

```diff
@@ -250,33 +251,76 @@ defmodule Facebook do
   """
   def objectCount(:comments, object_id, access_token) do
     "comments"
-      |> objectSummary(object_id, access_token)
+      |> getScopeData(object_id, access_token)
+      |> getSummary
       |> summaryCount
   end
 
+  @doc """
+  Gets the number of reactions that an object has.
+
+  Expected type of reactions:
+    * :haha
+    * :wow
+    * :thankful
+    * :wow
+    * :angry
+    * :love
+    * :none
+
+  ## Examples
+      iex> Facebook.objectCount(:reaction, :wow, "769860109692136_1173416799336463", "<Token>")
+      2
+      iex> Facebook.objectCount(:reaction, :haha, "769860109692136_1173416799336463", "<Token>")
+      12
+      iex> Facebook.objectCount(:reaction, :thankful, "769860109692136_1173416799336463", "<Token>")
+      33
+  """
+  def objectCount(:reaction, react_type, object_id, access_token)  when is_atom(react_type) do
+    type = react_type
+      |> Atom.to_string
+      |> String.upcase
+
+    params = [access_token: access_token, type: type, summary: "total_count"]
+    if !is_nil(Config.appsecret) do
+      params = params ++ [appsecret_proof: encrypt(access_token)]
+    end
+
+    Facebook.Graph.get(~s(/#{object_id}/reactions), params)
+      |> getSummary
+      |> summaryCount
+  end
+
+  """
+  Gets information related to the scope sent as argument.
+
+  Expected scopes:
+    * likes
+    * comments
+
+  See: https://developers.facebook.com/docs/graph-api/reference/object/comments
+  See: https://developers.facebook.com/docs/graph-api/reference/object/likes
+  """
+  defp getScopeData(scope, object_id, access_token) do
+    params = [access_token: access_token, summary: true]
+    if !is_nil(Config.appsecret) do
+      params = params ++ [appsecret_proof: encrypt(access_token)]
+    end
+
+    Facebook.Graph.get(~s(/#{object_id}/#{scope}), params)
+  end
```

Comment:
I'd actually simplify this function too by just specifying objectCount and just passing any scope to FB. Similar to what has been done with pageFeed